api/v1beta2/suite_test.go

```diff
@@ -27,3 +29,27 @@ func TestCmd(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "FDB v1beta2 API")
 }
+
+// sortedMapValues gets the values from a map as a list, sorted according to the ordering of the underlying keys.
+func sortedMapValues[K cmp.Ordered, V any](data map[K]V) []V {
```

Comment:
Do you think it makes sense to add a comment why we can't use those helper functions from the core library: https://pkg.go.dev/maps#Values?

Reply:
I took a look at those functions, and I think the main benefit to this approach is the consistent ordering for the HaveExactElements matcher. However, I think I can express what we actually want to check here more cleanly with a ConsistOf matcher, so I'll go that route instead.